tests/integration/test_s3_storage_conf_proxy/test.py

```diff
@@ -17,6 +17,22 @@ def run_resolver(cluster):
     )
     cluster.exec_in_container(container_id, ["python", "resolver.py"], detach=True)
 
+    for i in range(10):
```

Comment:
Please add a comment, something like:
verifying that the external resolver started and produces the expected results.